Functions/InputOutput/xASL_io_DcmtkRead.m

```diff
@@ -239,19 +235,55 @@
 	end
 end
 
+if isfield(header, 'Private_0043_1083')
+	header.AssetRFactor = header.Private_0043_1083;
```

Comment:
Note that this is a GE-specific parameter. I would move the vendor-specific parameters into vendor-specific sections for readability.